components/organisation-unit-tree/src/organisation-unit-node/organisation-unit-node.js

```diff
@@ -32,6 +32,115 @@ const LoadingSpinner = () => (
     </div>
 )
 
+const getFilteredChildren = ({ orgChildren, filter, node }) => {
+    if (!filter?.length) {
+        return orgChildren
+    }
+
+    return orgChildren.filter((child) => {
+        return isPathIncluded(filter, `${node.path}/${child.id}`)
+    })
+}
+
+export const OrganisationUnitNodeChildren = ({
```

Comment:
This file is getting rather large. Personally I'd extract this component to a separate file.